src/Orders/BusinessEmails/Responses/CreateResponse.php

```diff
@@ -6,12 +6,83 @@
 use Money\Money;
 use ResellerClub\Response;
 
+/**
+ * @see https://manage.resellerclub.com/kb/answer/2156
+ */
 class CreateResponse extends Response
 {
     /**
-     * Get the invoice ID parameter.
+     * Get the business email order id.
+     *
+     * @return int
+     */
+    public function orderId(): int
+    {
+        return $this->entityid;
+    }
+
+    /**
+     * Gets the domain name that this business email order has been created for.
+     *
+     * @return string
+     */
+    public function domain(): string
+    {
+        return $this->description;
+    }
+
+    /**
+     * Gets the action status for this business email order.
+     *
+     * @return string
+     */
+    public function actionStatus(): string
+    {
+        return $this->actionstatus;
+    }
```

Comment:
Do these `action*` parameters need to be [`Action`](https://github.com/Pitchero/ResellerClub/blob/master/src/Action.php) instances instead?